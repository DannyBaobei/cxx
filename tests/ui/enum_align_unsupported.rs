#[cxx::bridge]
mod ffi {
    #[repr(align(2))]
    enum Bad {
        A,
    }
}

fn main() {}
