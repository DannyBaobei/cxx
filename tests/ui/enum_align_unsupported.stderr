error: repr(align) on enums is not supported
 --> $DIR/enum_align_unsupported.rs:3:5
  |
3 | /     #[repr(align(2))]
4 | |     enum Bad {
5 | |         A,
6 | |     }
  | |_____^
